modules/remote.sh

```diff
@@ -50,7 +50,11 @@ function remote {
     fi
   elif [ "$option" = "commands" ]; then
     source $SCRIPTFOLDER/_treehouses && _treehouses_complete 2>/dev/null
-    echo "$every_command"
+    while IFS= read -r line;
+    do
+      cmd_str+="\"$line\","
+    done <<< "$every_command"
+    printf "{\"commands\":["%s"]}\n" "${cmd_str::-1}"
```

Comment:
@kjong it may be confusing but we do not want right now to change `treehouses remote commands` but we wanna add an additional command `treehouses remote commands json`